modules/ROOT/pages/access-recovery.adoc

```diff
@@ -0,0 +1,10 @@
+= Access Recovery
+
+If you've lost the private key of an SSH keypair used to log into Fedora CoreOS, and do not have any password logins set up to use at the console, you can gain access back to the machine using the `init=/bin/sh` trick:
+
+. When booting the system, intercept the GRUB menu and edit the entry to append `init=/bin/sh` to the kernel argument list, then press Ctrl-X to resume booting.
```

Comment:
By holding the left shift key or sequentially pressing the up and down arrows to interrupt the grub timeout, then press `e` to edit the selected entry, entering the grub user (usually root) and password if necessary. (Do we support grub passwords?)

Reply:
Maybe let's keep further tweaks as follow-ups and just get this one in? It does more good as is on the docs site, than not-quite-perfect sitting here. :)